functionnal_tests/cypress/integration/share/guest_upload_form__upload_file_with_password_spec.js

```diff
@@ -0,0 +1,50 @@
+let workspaceId
+const passwordForUploadLink = 'strongPassword'
+let guestUploadUrl
+
+describe('Guest upload page', () => {
+  before(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('administrators')
+    cy.fixture('baseWorkspace').as('workspace').then(workspace => {
+      workspaceId = workspace.workspace_id
+      cy.createGuestUploadLink(workspaceId, ['dummyemail@nothingtodo.here'], passwordForUploadLink)
+        .then(response => {
+          if (response.status !== 200) throw Error('failed to create guest upload link')
+          guestUploadUrl = response.body[0].url
+        })
+    })
+    cy.logout()
+  })
+
+  beforeEach(function () {
+    cy.loginAs('administrators')
```

Comment:
this is only required in the before()
Even though it doesn't change anything here, I think it's better to ensure that the guest upload works when not logged in since it'll probably be the most used use case

Reply:
good remark, I'll remove it